src/main/java/net/mcreator/generator/Generator.java

```diff
@@ -200,7 +200,7 @@ public boolean generateElement(GeneratableElement element) {
 			this.generateElement(element, true);
 			return true;
 		} catch (TemplateGeneratorException e) {
-			e.printStackTrace();
+			LOG.error("Failed to generate mod element: " + element.getModElement().getName(), e);

```

Comment:
I'm sorry for changing this on a wrong pull request, but shouldn't this line look like it does now?

Reply:
Yes, direct stack trace print is not OK